turbinia/workers/__init__.py

```diff
@@ -152,6 +152,14 @@ def close(self, task, success, status=None):
       success: Bool indicating task success
       status: One line descriptive task status.
     """
+
+    # Calls postprocess on output evidence types that never
+    # TODO (wyassine): come up with a better way to call this as
+    # this may cause some issues with future evidence types in
+    # which postprocess should not be called.
+    for out_evidence in self.evidence:
+      out_evidence.postprocess()
```

Comment:
I think you can move this down to line 178 because it's already looping over the input evidence there.